udata/migrations/2024-01-29-fix-reuse-and-dataset-with-private-None.py

```diff
@@ -0,0 +1,36 @@
+'''
+Some old dataset/reuse don't have a `private` field. It should be False by default.
+'''
+import logging
+
+from udata.models import Reuse, Dataset
+
+log = logging.getLogger(__name__)
+
+
+def migrate(db):
+    log.info('Processing Reuse.')
+
+    reuses = Reuse.objects().no_cache().timeout(False)
```

Comment:
We can filter on private=None
```suggestion
    reuses = Reuse.objects(private=None).no_cache().timeout(False)
```